apps/user/src/user/entities/user.entity.ts

```diff
@@ -15,15 +15,15 @@ export class User {
   @Column('simple-array')
   public stack_category_id: number[];
 
-  @ManyToOne(() => PositionCategory)
+  @ManyToOne(() => PositionCategory, { eager: true })
   @JoinColumn({ name: 'position_category_id' })
   public position_category_id: PositionCategory;
 
-  @ManyToOne(() => ProjectCategory)
+  @ManyToOne(() => ProjectCategory, { eager: true })
   @JoinColumn({ name: 'project_category_id' })
   public project_category_id: ProjectCategory;
```

Comment:
```suggestion
 @Column('simple-array')
  public project_category_id: number[];
```

기획의도가 반영되면 해당 엔티티도 simple-array의 형식으로
number 배열을 저장하도록 변경 되어야 할 것 같습니다.